Improve descriptions tag (#2772)
near/docs: docs/smart-contracts/anatomy/actions.md

---
id: actions
title: Transfers & Actions

> description: "Learn how NEAR smart contracts perform actions like transferring NEAR, calling contracts, creating sub-accounts, deploying contracts, and managing access keys"

Reviewer: ```suggestion
description: "Learn how contracts can make transfers, call other contracts, and more"
```
